deploy.tex

```diff
@@ -1,50 +1,48 @@
-Early on we adopted Kubernetes which provided a service architectures that is well isolated from the underlying infrastructure.
+Being cloud-ready is certainly the biggest challenge for most projects/applications which intend to leverage the commercial cloud.
+Early on we adopted Kubernetes which provided a service architecture that is well isolated from the underlying infrastructure.
 This approach has already paid off massive dividends.
 For example when funding lines suddenly shifted we were able to painlessly transition from an on-premises facility to an Interim Data Facility on Google Cloud.
 Furthermore the Rubin Science Platform (RSP) became a generic data services platform that is currently deployed on eight distinct (and distinctly managed) infrastructures (on-prem and cloud).
-Finally this has allowed us to leverage the Cloud for services like the RSP which benefit from its advantages such as elasticity, scalability, isolation.
+Finally this has allowed us to leverage the cloud for services like the RSP which benefit from its advantages such as elasticity, scalability, isolation.
 
 
 
 
 \subsection{Division of responsibilities}
-We have had ways to abstract system services from our developed services for some time, containers and java effectively allow one to run \emph{anywhere}.
+We have had ways to abstract system services from our developed services for some time, containers and Java effectively allow one to run \emph{anywhere}.
 These work well enough for single user single processor deployments.
-When you need to scale up another level of orchestration is needed, kubernetes fits neatly between our infrastructure and our developed applications providing a powerful container orchestration and resource management system.
+When scaling up to another level of orchestration, Kubernetes fits neatly between our infrastructure and our developed applications providing a powerful container orchestration and resource management system.
 
-Our agreement with each facility is to provide a Kubernetes deployment platform upon which we may deploy our services.
-Each thus provides a kubernetes environment with disk, network and compute resources upon which we spin up our services.
-A vault for secrets is also needed.
+Our agreement with each facility is to provide a Kubernetes deployment platform upon which we may deploy our services and our secrets vault for security.
+Each thus provides a Kubernetes environment with disk, network and compute resources upon which we spin up our services.
 From the service perspective each facility looks similar.
 It feels like the first time that this actually works well - we have learned to make our applications portable of course but also the technology is so much better than any precursors.
 
 \subsection{Phalanx, Helm and ArgoCD}
 
-Helm\footnote{https://helm.sh/} is a standard approach to tell Kubernetes about software.
```

Comment:
I wonder if this should stay here and remove it from the earlier sect
ion so all the tech footnotes are together.